azerothcore/azerothcore-wotlk
# refactor(Core/Scripts): Simplify speeches for Special Surprise quests.

Co-Authored-By: Tec Dian de Akilet <12619865+TecDian@users.noreply.github.com>
Co-Authored-By: Aokromes <Aokromes@users.noreply.github.com>
Cherry-picked from: https://github.com/TrinityCore/TrinityCore/commit/3f34c2f5ebe8d0d9f97b169d1ace07d6ab3f3102.



## Changes Proposed:
- Replaces existing `creature_text` entries with ones specific to each creature.
- Removes massive switch tailored for each race, replacing with one that applies to all races.

## Issues Addressed:

- Closes unreported issue.

## SOURCE:

N/A

## Tests Performed:

- Untested.

## How to Test the Changes:

1. Add the various quests according to your race.
2. Test the interaction for the quest.
3. Make sure it plays out without issues.


## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
